js/addcss.mjs

```diff
@@ -1,3 +1,11 @@
+/**
+ * Adds a CSS file to the document head if it is not already present.
+ *
+ * @param {string} url - The URL of the CSS file to add.
+ * @param {string} [nameToCheck] - The name pattern to check in the URL. If provided,
+ *    the CSS file will only be added if the pattern is found in the URL.
```

Comment:
This is only part of the logic of the `nameToCheck` parameter. It should read "A regular expression pattern to check for in existing CSS links in the document, used to avoid duplicating CSS files. If the pattern is found in the document, the new link is not added. For consistency, the pattern is also checked against the url parameter to make sure it will exist in the document after adding the new link. If the pattern is not found in the url, an error is logged to the console and the file is not added."